ui/app/components/app-item/deployment.hbs

```diff
@@ -7,29 +7,66 @@
   <hr>
   <LinkTo
     data-test-deployment-list-item
-    title={{ concat
-      "v" @deployment.sequence ": "
-      (t (concat "page.deployments.status_prefix." @deployment.status.state))
-      (component-name @deployment.component.name)}}
+    title={{concat (t "page.deployment.title") " v" @deployment.sequence }}
     @route="workspace.projects.project.app.deployment.deployment-seq"
     @models={{array @deployment.sequence}}
   >
     <b class="badge badge--version">v{{@deployment.sequence}}</b>
-    <OperationStatusIndicator::Deployment @operation={{@deployment}}>
-      <:small-text>
-        {{#if (not-eq @latest.id @deployment.id)}}
-        <span class="replacement-info">
-          {{t "page.deployments.replaced_label"}}<b class="badge badge--version">v{{@latest.sequence}}</b>
+    <div class="app-item__deployment-state">
+      {{!-- ROW: status report + deployment url --}}
+      {{#if @deployment.statusReport.health}}
+        {{#let (hash
+          deploymentHealth=@deployment.statusReport.health.healthStatus
+          urlOrStatus=(or
+            (clean-up-url @deployment.preload.deployUrl)
+            (t (concat "status_report_indicator.label."
+              (downcase @deployment.statusReport.health.healthStatus))
+            )
+          )
+        ) as |vars|}}
+          <span
+            data-test-health-status={{downcase vars.deploymentHealth}}
+            class="deploy-url__status deploy-url__status--{{downcase vars.deploymentHealth}}"
+          />
```

Comment:
<img width="334" alt="CleanShot 2022-01-11 at 17 27 02@2x" src="https://user-images.githubusercontent.com/34030/148981974-bc5845a9-3822-4295-b8a5-f32066a9f0e2.png">

When a URL is present, it looks like we’re only using the color of the indicator to convey the health status. We should add an additional means to convey this information.

Per [WCAG SC 1.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html):

> Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element

Reply:
Just to hammer the point home, here’s how a simulation of how this would look to someone with deuteranopia (via [Sim Daltonism](https://michelf.ca/projects/sim-daltonism/)):

<img width="348" alt="CleanShot 2022-01-11 at 17 41 46@2x" src="https://user-images.githubusercontent.com/34030/148984449-9432902e-7c0e-4384-b995-0fb7e240ef71.png">

Reply:
@sabrinako How about we show the URL if and only if the health status is `available`? I think that would adhere pretty closely to the WCAG guidelines, on the basis that the presence of a URL implies availability.

Reply:
@gregone pointed out that we might want to also show the URL in the `unknown` state, for deployments that don’t/can’t provide health status. I think this is worth digging into, but perhaps in a subsequent PR.